### PR TITLE
SERVER-59815 update readme

### PR DESCRIPTION
**Pull request by brushless-glitch**
This merges selinux.md previously targeted to the base repo into README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,19 @@
 
 This is an official SELinux policy for MongoDB server
 
+Security-Enhanced Linux (SELinux) is an implementation of mandatory cccess controls (MAC)
```
Comment (spencerjackson) [inline README.md]: s/mandatory cccess controls/mandatory access control/

File: README.md [hunk 2]
```diff
@@ -2,6 +2,19 @@
...
+in the Linux kernel, checking for allowed operations after standard discretionary access
+controls (DAC) are checked.
+
+## Scope
+
+* policies apply to computers running RHEL7 and RHEL8 only
+* cover standard mongodb-server systemd based installations only
+* both community and enterprise versions are supported
+
+Supplied policies do not cover any daemons or tools other than mongod, such as: sharding,
```
Comment (spencerjackson) [inline README.md]: s/sharding/mongos/

File: README.md [hunk 3]
```diff
@@ -26,3 +39,47 @@ sudo make install
...
+
+- daemon binary is installed in `/usr/bin/mongod`
+- database is located in `/var/lib/mongo`
+- log file must be located in `/var/log/mongodb/`
+- runtime data (PID) should be in `/var/run/mongodb/` or `/run/mongodb/`. On RHEL systems
+`/var/run` is a symbolic link to `/run`. This should not be changed
+- default unix socket file goes to `/tmp`, which must stay a default location provided by
+operating system. It can not be a symbolic link to another location
+- default user created and configured by installer is used to run service
+- systemd unit file `/usr/lib/systemd/system/mongod.service` created by installer is used
```
Comment (spencerjackson) [inline README.md:57]: Can an unconfined user run `/usr/bin/mongod`?

**Review by spencerjackson — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (brushless-glitch) [reply README.md:57]: Should. We only transition mongod to mongod_t domain when launched from systemd:

```
type_transition initrc_domain mongod_exec_t:process mongod_t;
```

File: README.md (hunk 1 above)
Comment (brushless-glitch) [reply README.md]: 🤦... Done

File: README.md (hunk 2 above)
Comment (brushless-glitch) [reply README.md]: Done

**Review by spencerjackson — APPROVED** [review]
LGTM

**Pull request merged**